src/dataflow/components/dataflow-program.tsx

```diff
@@ -195,6 +197,15 @@ export class DataflowProgram extends BaseComponent<IProps, IState> {
     }
   }
 
+  private getEditorStyle = () => {
+    const style: React.CSSProperties = {};
+    const documentElts = document.getElementsByClassName("document-content");
+    const documentElt: HTMLInputElement = documentElts && (documentElts[0] as HTMLInputElement);
+    const topbarHeight = 50;
```

Comment:
This seems fragile. How about finding the top bar element and querying it for its height? The class-name seems less likely to change than the pixel height.

Reply:
Agreed.  Fixed.